packages/node_modules/@ciscospark/plugin-logger/src/config.js

```diff
@@ -2,18 +2,22 @@
  * Copyright (c) 2015-2017 Cisco Systems, Inc. See LICENSE file.
  */
 
+/**
```

Comment:
I like these defined up here, makes it much easier to read.

Reply:
haha, it wasn't really by choice; it was the only way i could get the docs generator to nest things in anyway approaching something reasonable.